internal/util/util.go

```diff
@@ -65,16 +66,39 @@ func GetTempDir() string {
 	return "/var/tmp"
 }
 
+// ExportImage copies contents of the image to a new location
+func ExportImage(store storage.Store, opts define.BuildOutputOption) error {
+	if !opts.Push {
+		return nil
```

Comment:
You're right, `type=image` on its own doesn't require anything additional since the image is built locally before we get to exports. So now, I'm handling a situation where we have `type=image,push=true` and if not, then proceed further down.